cookbooks/sidekiq6/templates/default/sidekiq.service.erb

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Sidekiq Worker <%= @count %>
+
+
+[Service]
+Type=notify
+WorkingDirectory=/data/<%= @app_name %>/current
+ExecStart=/bin/bash -l -c "bundle exec sidekiq -e <%= @rails_env %> -C /data/<%= @app_name %>/current/config/sidekiq_<%= @count %>.yml"
```

Comment:
There is an issue with systemd where after some time (we guessed it was the size of the log file) when restarting the processes, the logs are not appended to the file but it overwrites it causing a mess.
The issue was explained here: https://newbedev.com/redirect-systemd-service-logs-to-file
To fix the append in our log files as we keep the logs, we had to update the command line:
```suggestion
ExecStart=/bin/bash -lc "exec bundle exec sidekiq -e <%= @rails_env %> -C /data/<%= @app_name %>/current/config/sidekiq_<%= @count %>.yml >>/data/<%= @app_name %>/current/log/sidekiq_<%= @count %>.log 2>>/data/<%= @app_name %>/current/log/sidekiq_<%= @count %>.stderr.log"
```